config/owasp/suppressions.xml

```diff
@@ -31,4 +31,56 @@
     <cve>CVE-2016-5394</cve>
     <cve>CVE-2016-6798</cve>
   </suppress>
+  <suppress>
```

Comment:
Are all these suppressions justified?

Reply:
it is coming wiremock-standalone-2.20.0.jar and it is used by orchestrator and suppressions are same as orchestrator.

Reply:
These are lot of suppressions. Can we not upgrade the jar version or if no fix is available can we add a comment ?

Reply:
exclude wiremock-standalone  2.20 and used 'com.github.tomakehurst', name: 'wiremock-jre8', version: '2.25.0' ,looks fine for now